lib/clixon/clixon_err.h

```diff
@@ -43,6 +43,8 @@
 #ifndef _CLIXON_ERR_H_
 #define _CLIXON_ERR_H_
 
+#include <cligen/cligen_buf.h>
```

Comment:
This violates the coding guidelines: https://github.com/clicon/clixon/blob/master/CONTRIBUTING.md#include-files:
> Avoid include statements in .h files, place them in .c files whenever possible.

Reply:
Reverted.